cmd/test/runner.go

```diff
@@ -64,7 +64,8 @@ func main() {
 			executor,
 			mc.NewDefaultFileResolver(),
 		)
-		err = runner.RunSingleJSONScenario(jsonFilePath)
+		traceGas := false
+		err = runner.RunSingleJSONScenario(jsonFilePath, traceGas)
```

Comment:
Passing `traceGas` will override whatever the JSON scenario contains. Don't pass it at all.